AmplifyPlugins/API/AWSAPICategoryPluginIntegrationTests/GraphQL/GraphQLWithUserPoolIntegrationTests/AuthDirective/GraphQLAuthDirectiveIntegrationTests.swift

```diff
@@ -0,0 +1,244 @@
+//
+// Copyright 2018-2020 Amazon.com,
+// Inc. or its affiliates. All Rights Reserved.
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+
+import XCTest
+@testable import Amplify
+import AWSMobileClient
+import AWSAPICategoryPlugin
+@testable import AWSAPICategoryPluginTestCommon
+@testable import AmplifyTestCommon
+import AWSPluginsCore
+
+class GraphQLAuthDirectiveIntegrationTests: XCTestCase {
+
+    static let amplifyConfiguration = "GraphQLAuthDirectiveIntegrationTests-amplifyconfiguration"
+    static let awsconfiguration = "GraphQLAuthDirectiveIntegrationTests-awsconfiguration"
+    static let credentials = "GraphQLAuthDirectiveIntegrationTests-credentials"
+    static var user1: String!
+    static var user2: String!
+    static var password: String!
```

Comment:
Is there a strong argument/reason to make these static?  Reading this make me believe that the tests in this file depend on each other and, I don't think that we want that.... Instead, i think we want independent test that we can run in isolation.

Reply:
no strong reason, making these instances and removing static init made accessing these variables much easier, without having to put the class name in front. having them static meant we only read the credentials file once

Reply:
I generally like to avoid static variables for unit tests.  variables can be accidentally overwritten and then impact other tests..  If it's not a lot of work can we switch this over to non-static?

Reply:
ah yes, should have been more clear, i have already updated to use instance variables